src/main/java/org/cryptomator/ui/vaultoptions/VaultOptionsController.java

```diff
@@ -42,11 +43,11 @@ public void initialize() {
 		window.setOnShowing(this::windowWillAppear);
 		selectedTabProperty.addListener(observable -> this.selectChosenTab());
 		tabPane.getSelectionModel().selectedItemProperty().addListener(observable -> this.selectedTabChanged());
-		var vaultScheme = vault.getVaultConfigCache().getUnchecked().getKeyId().getScheme();
-		if(!vaultScheme.equals(MasterkeyFileLoadingStrategy.SCHEME)){
+		var vaultKeyLoader = vault.getVaultSettings().lastKnownKeyLoader.get();
+		if(!vaultKeyLoader.equals(MasterkeyFileLoadingStrategy.SCHEME)){
```

Comment:
This check can also be moved to a static method inside the KeyLoadingStrategy (similar to the `isHubVault` method)